pkgs/development/python-modules/dfdiskcache/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pandas
+, simplesqlite
+, typing-extensions
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "df-diskcache";
+  version = "0.0.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "thombashi";
+    repo = "df-diskcache";
+    rev = "v${version}";
+    hash = "sha256-s+sqEPXw6tbEz9mnG+qeUSF6BmDssYhaDYOmraFaRbw=";
+  };
+
+  propagatedBuildInputs = [
+    pandas
+    simplesqlite
+    typing-extensions
+  ];
+
+  preCheck = ''
+    # Needed for Permission denied: '/homeless-shelter'
+    export HOME=$(mktemp -d)
+  '';
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
```

Comment:
Also missing pythonImportsCheck and running tests https://github.com/thombashi/df-diskcache/tree/master/tests

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/299429/commits/672238a89d2b499612ece9dcf53df01ab9d6ad19